### PR TITLE
feat(env): add brovider env

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -11,3 +11,4 @@ KEYCARD_SECRET=
 # Sentry (Optional)
 SENTRY_DSN=
 SENTRY_TRACE_SAMPLE_RATE=
+BROVIDER_URL=https://rpc.snapshot.org
```
Comment (ChaituVR) [inline .env.example:14]: Maybe better to have it as a const variable?

Easy to find/maintain it from code than from env. as it is not often changed

Comment (wa0x6e) [reply .env.example:14]: Or use both method, keep in .env so we can still override it via .env var (such as when using with Docker), and redefine locally on top of file, to set a default value 

`const broviderUrl = process.env.BROVIDER_URL  || 'https://rpc.snapshot.org'`

Leaving the .env var optional

File: src/scores.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@ import { get, set } from './aws';
 import { getBlockNum, sha256 } from './utils';
 import serve from './requestDeduplicator';
 
+const broviderUrl = process.env.BROVIDER_URL || 'https://rpc.snapshot.org'
```
Comment (ChaituVR) [inline src/scores.ts]:
```suggestion
const broviderUrl = process.env.BROVIDER_URL || 'https://rpc.snapshot.org';
```

File: src/utils.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import snapshot from '@snapshot-labs/strategies';
 import { createHash } from 'crypto';
 
+const broviderUrl = process.env.BROVIDER_URL || 'https://rpc.snapshot.org'
```
Comment (ChaituVR) [inline src/utils.ts]:
```suggestion
const broviderUrl = process.env.BROVIDER_URL || 'https://rpc.snapshot.org';
```

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**